pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -43,6 +43,20 @@ self: super: {
     hinotify = if pkgs.stdenv.isLinux then self.hinotify else self.fsnotify;
   };
 
+  fsnotify = overrideCabal (super.fsnotify.override {
+    hinotify = if pkgs.stdenv.isLinux then self.hinotify else self.hfsevents;
+  }) (drv: {
+    doCheck = !pkgs.stdenv.isDarwin;
+  });
+
+  system-fileio = overrideCabal super.system-fileio (drv: {
+    doCheck = !pkgs.stdenv.isDarwin;
```

Comment:
Ditto. It would be nice to have these issues fixed and/or to have some URL to a ticket describing the issue.